### PR TITLE
Add new onboarding tour. (#497)

### PR DESCRIPTION
**Pull request by pocmo**
The layouts are not final yet - but I want to land this state to get the strings in (and also the images to get closer to the final APK size)

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/firstrun/FirstrunPagerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.mozilla.focus.R;
+
+public class FirstrunPagerAdapter extends PagerAdapter {
+    private static int[] PAGE_LAYOUTS = {
```
Comment (ahunt) [inline app/src/main/java/org/mozilla/focus/firstrun/FirstrunPagerAdapter.java]: nit: final

File: app/src/main/java/org/mozilla/focus/utils/Settings.java [hunk 2]
```diff
@@ -39,14 +39,14 @@ private Settings(Context context) {
 
     public boolean shouldBlockImages() {
         // Not shipping in v1 (#188)
-        /* return preferences.getBoolean(
+        return preferences.getBoolean(
```
Comment (ahunt) [inline app/src/main/java/org/mozilla/focus/utils/Settings.java]: Are we reenabling image blocking, or is this accidental?

File: app/src/main/java/org/mozilla/focus/utils/Settings.java [hunk 3]
```diff
@@ -39,14 +39,14 @@ private Settings(Context context) {
...
                 resources.getString(R.string.pref_key_performance_block_images),
-                false); */
-        return false;
+                false);
     }
 
     public boolean shouldShowFirstrun() {
-        return !preferences.getBoolean(FirstrunFragment.FIRSTRUN_PREF, false);
+        return true;
```
Comment (ahunt) [inline app/src/main/java/org/mozilla/focus/utils/Settings.java]: I'm guessing this was for testing : )

Comment (pocmo) [inline app/src/main/java/org/mozilla/focus/utils/Settings.java]: Yeah, I wanted to undo it and apparently did it for blocking images :D

**Pull request merged**